### ISSUE TITLE
Figure out how to calculate a gradient norm for a training step

### ISSUE THREAD

**Issue by Themightyfirefly** [Themightyfirefly/jl_cockpit#9]
The training returns the loss and something called grads. We must figure out how to calculate the norm out of grads and then append it to the grad_norms list

**Comment by Themightyfirefly** [comment]
I looked into it and I have absolutely no idea. I just cant understand how to get the gradients from the grads variable. I added some information in the code, the return value seems to be some directory like structure, but I don't know how to access it. We should ask Adrian tomorrow
